src/std/os/signal-handler.ss

```diff
@@ -130,6 +130,7 @@
                         (when event-handler
                           (signal-handler-dispatch event-handler))))
                     (raise-os-error signal-handler-wait
+                                    EIO
```

Comment:
???

Reply:
Well, `raise-os-error` expects an `errno` argument here. I don't know what value to put here, but it has to be an `errno`, if possible one that makes sense in this context (I admit I don't understand the kqueue protocol, so I can't say what would fit).

Reply:
ok, fair enough.